lab_1/main.py

```diff
@@ -74,10 +137,28 @@ def detect_language(unknown_profile: dict, profile_1: dict, profile_2: dict, top
     :param top_n: a number of the most common words
     :return: a language
     """
-    pass
+    if (not isinstance(unknown_profile, dict)
+            or not isinstance(profile_1, dict)
+            or not isinstance(profile_2, dict)
+            or not isinstance(top_n, int)):
+        return None
+    # use function compare_profiles
+    share_the_first_language = compare_profiles(unknown_profile, profile_1, top_n)
+    share_the_second_language = compare_profiles(unknown_profile, profile_2, top_n)
+    # detect the language via share of common tokens
+    if share_the_first_language == share_the_second_language:
+        if_two_language = sorted([profile_1["name"], profile_2["name"]])
+        language_name = if_two_language[0]
```

Comment:
can you get [0] just here? `sorted([profile_1["name"], profile_2["name"]])`

Reply:
Done